web-client/src/presenter/sequences/CaseAssociation/validateEditPetitionerCounselSequence.ts

```diff
@@ -12,7 +13,7 @@ export const validateEditPetitionerCounselSequence = [
       setRepresentingFromRepresentingMapActionFactory('form'),
       validateEditPetitionerCounselAction,
       {
-        error: [setValidationErrorsAction],
+        error: [setValidationErrorsAction, setValidationAlertErrorsAction],
```

Comment:
Here and throughout, the `setValidationAlertErrorsAction` is keeping our `ErrorNotification` consistent with our inline messages.